.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+
+Here are a few tips to increase the likelihood of being merged:
+
+- [ ] Write tests.
+- [ ] Write a [good commit message](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
+- [ ] Allow [edits from maintainers](https://blog.github.com/2016-09-07-improving-collaboration-with-forks/).
+
+### API Breaking changes
+
+We care deeply about backward compatibility for our API clients libraries. If it's necessary,
+we're ready to break backward compatibility but this should be pretty rare.
+
+If you want to make breaking changes, open an issue first to discuss it with the maintainers.
+
+## Label caption
+
+Labels across all repositories are normalized.
+
+<!-- DO NOT EDIT THE TABLE NOR REMOVE HTML COMMENTS -->
+<!-- LABELTABLE -->
+| Label                                                                    | Meaning                                                                                |
+|--------------------------------------------------------------------------|----------------------------------------------------------------------------------------|
+| ![#050f2c](https://placehold.it/15/050f2c/000000?text=+) Do not merge    | PR should not be merged (decided by maintainers)                                       |
+| ![#ffc168](https://placehold.it/15/ffc168/000000?text=+) WIP             | PR is not ready, no need to look at it (decided by contributors)                       |
+| ![#2ede98](https://placehold.it/15/2ede98/000000?text=+) Ready           | The PR is ready, reviewed, tests are green, if you're brave enough: click merge button |
+| ![#ffc168](https://placehold.it/15/ffc168/000000?text=+) Waiting for API | The feature is implemented but the REST API is not live yet                            |
+| ![#3369e6](https://placehold.it/15/3369e6/000000?text=+) Discussion      | We need everyone's opinion on this, please join the conversation and share your ideas  |
+| ![#3369e6](https://placehold.it/15/3369e6/000000?text=+) Support         | A user needs help but it's not really a bug                                            |
+| ![#3369e6](https://placehold.it/15/3369e6/000000?text=+) API feature     | New API feature added to every client (like query rules)                               |
+| ![#3369e6](https://placehold.it/15/3369e6/000000?text=+) Chore           | CI, docs, and everything around the code                                               |
+| ![#ff4f81](https://placehold.it/15/ff4f81/000000?text=+) Bug             | It's a bug, fix it!                                                                    |
+| ![#b60205](https://placehold.it/15/b60205/000000?text=+) Breaking change | RED ALERT! This means we need a new major version                                      |
+| ![#ff6c5f](https://placehold.it/15/ff6c5f/000000?text=+) Easy Pick       | If you want to contribute, this one is _easy_ to tackle!                               |
```

Comment:
As mentioned in the slack discussion when you introduced these labels, I would rather use GitHub's [Default Label](https://help.github.com/articles/about-labels/#using-default-labels) for this, `good first issue`. GitHub leverages it in their UI to encourage beginners to contribute, [which prompted eslint](https://github.com/eslint/eslint/issues/9609) to rename their similar label to follow the convention.

Is there any specific reason for which we use a custom label instead of the default? 🤔

Reply:
Would have opened an issue on `algoliasearch-client-common`, but weirdly I cannot open any issue. Can you check the setting? If we tell external contributors that `any Pull Request there is welcome`, we maybe should make it open :trollface:

Reply:
Pull request are welcome, but not issues :D haha I turned them off to see how it goes. I'd rather leave pull requests only ;)

Reply:
I get you 😛 ! However, how should we algolians report issues with common files? Open an issue on the first repo we see the problem and let you centralize them all? 🤔

Reply:
Also, I would still like your take on [my first comment](https://github.com/algolia/algoliasearch-django/pull/247#discussion_r176050342) 🙃

Reply:
Sorry! About your first comment, I will rename "easy pick" to "good first issue". I'm not sure how they leverage it in their UI, but it's clearer and it would be great if it could become a standard.

About issues, I was secretly hoping people would fix things directly but I think I should open  the issue section.

Reply:
>About issues, I was secretly hoping people would fix things directly but I think I should open the issue section.What you

Reply:
> I'm not sure how they leverage it in their UI

When I opened the `Labels` setting page for the first time, I saw this:
- >**Add labels for new contributors**
  > Now, GitHub will help potential first-time contributors discover issues labeled 
  with `help wanted` or `good first issue`

----

  > About issues, I was secretly hoping people would fix things directly but I think I should open the issue section.

You can tell `please fix directly if it's a no brainer` in that repo's Issue Template :)